### ISSUE TITLE
Add SSH Options in Dashboard for Deployments

### ISSUE THREAD

**Issue by Mik-TF** [threefoldtech/tfgrid-sdk-ts#2065]
# Feature Request

- Add SSH Options in Dashboard for Deployments
  - add ssh pub key in authorized_keys
  - replace current ssh pub key with new key 

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/77026219/525f0831-d2b8-418b-8343-b7f8e6aea488)

# In Practice

It would be nice if users could add an ssh public key in the authorized_keys on their VMs, by using the SSH section of the dashboard. So we would add an option to update the authorized_keys file of all of the current user VM.

Also, it would be nice if users could update the ssh public key in the SSH section and that it would be reflected in all the VMs of the user (update existing pub key, not just adding a new key)

# Use

This is very useful for users that actively use the grid on perhaps more than just 1 computer.

**Comment by xmonader** [comment]
The thing for that to be done completely from the client side, this requires tcp sockets in the browsers, which is intentionally not allowed, for security reasons. What I've been thinking of, is the user can deploy another workload e.g gridagent that can some specific tasks and that one can do ssh manipulation or whatever things that aren't allowed from the browser

**Comment by Mik-TF** [comment]
I totally agree with you here. So perhaps the issue should be: Create Weblet to Interact with TFGrid (change/update ssh keys, make backups of data).
Let me know if I can help around this.

In the same vein, @scottyeager and I discussed of a weblet that could do automatic backups of other weblets on the playground.
Imagine you have a wordpress weblet running and you can use a "Backup Weblet" on the TF Playground that makes a QSSS version of your wordpress database. That could be nice. For this use case, the weblet could be set with mimimum CRU and MRU, and just focus on storage, so the backup is cheaper than running the whole deployment.
